css/styles.css

```diff
@@ -0,0 +1,292 @@
+/*the title of blog */
+

```

Comment:
Good Indentation :
Your code looks very clean and easy to read, thanks! Using indentations really makes a huge difference in the code readability. :D